main.gms

```diff
@@ -814,17 +813,17 @@ parameter
+  cm_iterative_target_adj   "settings on iterative adjustment for CO2 tax based on in-iteration emission or forcing level. Allow iteratively generated endogenous global CO2 price under peak budget constraint."
 ;
-  cm_iterative_target_adj  = 0;      !! def = 0  !! regexp = 0|2|3|4|5|6|7|9
-*' * (0): no iterative adjustment
-*' * (2): iterative adjustment II based on magicc calculated forcing (for both budget and tax runs), see modules/ 0 /magicc/postsolve.gms for direct algorithms of adjustment
-*' * (3): [requires 45_carbonprice = "NDC" and emiscen = 9] iterative adjustment III for tax based on 2025 or 2030 regionally differentiated emissions, see module/45_carbonprice/NDC/postsolve.gms for algorithm of adjustment
-*' * (4): iterative adjustment IV for both budget and tax runs based on CO2 FF&I emissions 2020-2100, see core/postsolve.gms for direct algorithms of adjustment
-*' * (5): iterative adjustment V for both budget and tax runs based on CO2 emissions 2020-2100, see core/postsolve.gms for direct algorithms of adjustment
-*' * (6): iterative adjustment VI for both budget and tax runs based on peak CO2 emissions budget, without changing temporal profile (i.e. with overshoot), see core/postsolve.gms for direct algorithms of adjustment
-*' * (7): iterative adjustment VII for tax runs based on peak CO2 emissions, with change of temporal tax profile after time of peak budget, aiming for net-zero thereafter, see core/postsolve.gms for direct algorithms of adjustment
-*' * (9): [REMIND 2.1 default for validation peakBudget runs, in combination with carbonprice = none; after the peaking year annual increase by c_taxCO2inc_after_peakBudgYr. Automatically shifts c_peakBudgYr to find the correct peaking year for a given .
+  cm_iterative_target_adj = 0;      !! def = 0  !! regexp = 0|2|3|4|5|6|7|9
+*' * (0): no iterative adjustment of CO2 price (terminology: CO2 price and CO2 tax in REMIND is used interchangeably)
+*' * (2): iterative adjustment of CO2 tax based on climate forcing calculated by climate model magicc (for runs with budget and/or CO2 tax constraints), see ./modules/15_climate/magicc/postsolve.gms for direct algorithm
+*' * (3): [requires 45_carbonprice = NDC and emiscen = 9] iterative adjustment of CO2 tax based on 2025 or 2030 regionally differentiated emissions, see ./modules/45_carbonprice/NDC/postsolve.gms for direct algorithm
+*' * (4): iterative adjustment of CO2 tax based on CO2 FF&I emission budget (i.e. without landuse emissions) (2020-2100), see core/postsolve.gms for direct algorithms
+*' * (5): iterative adjustment of CO2 tax based on economy-wide CO2 emissions (2020-2100), see core/postsolve.gms for direct algorithms
+*' * (6): iterative adjustment of CO2 tax based on economy-wide CO2 emission peak budget, features: without changing temporal profile (i.e. with overshoot), see core/postsolve.gms for direct algorithms
```

Comment:
What does "without changing temporal profile (i.e. with overshoot)" mean? We never change the temporal profile of the CO2 tax, that's valid for all realizations. As this is a peak budget realization, there is no overshoot.

Reply:
hmmm, I simply copied over the original text from main.gms https://github.com/remindmodel/remind/blob/develop/main.gms#L825C1-L825C16, I also don't know what the temporal profile or overshoot comments are referring to... do you know who wrote this originally? Even though I don't know what it means, it seems to be important, as it is the only thing that differs from setting=7 (I believe?)

how should I best describe 6 or 7?